lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -187,10 +194,40 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             });     
         } else {
             this.logger.verbose("acquireTokenSilent - attempting to acquire token from web flow");
+
             const silentCacheClient = this.createSilentCacheClient(request.correlationId);
             const silentRequest = await silentCacheClient.initializeSilentRequest(request, account);
-            result = silentCacheClient.acquireToken(silentRequest).catch(async () => {
-                return this.acquireTokenByRefreshToken(silentRequest);
+
+            // set the request's CacheLookupPolicy to Default if it was not optionally passed in
+            request.cacheLookupPolicy = (request.cacheLookupPolicy === undefined) ? CacheLookupPolicy.Default : request.cacheLookupPolicy;
```

Comment:
This is typically done in `initializeSilentRequest`: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/src/interaction_client/SilentCacheClient.ts#L70

Reply:
Also, it should treat any falsey value (null, undefined) as being the default:

```suggestion
            request.cacheLookupPolicy = request.cacheLookupPolicy || CacheLookupPolicy.Default;
```

Reply:
@jasonnutter `initializeSilentRequest` returns a `Promise<CommonSilentFlowRequest>` (msal-common), not a `Promise<SilentRequest>` (msal-browser). Therefore, I would need to copy/paste the suggested change above to line 66 in SilentCacheClient.ts. If we did that, then we could refactor the code in msal-browser and would no longer need to pass around the silentRequest from PublicClientApplication to ClientApplication. Thoughts?

Reply:
Given that, I think this is fine here but please create a new request object rather than modifying the object passed in.